CppSamples/Maps/ShowDeviceLocationUsingIndoorPositioning/ShowDeviceLocationUsingIndoorPositioning.cpp

```diff
@@ -79,11 +83,49 @@ void ShowDeviceLocationUsingIndoorPositioning::setMapView(MapQuickView* mapView)
   m_mapView = mapView;
   m_mapView->setMap(m_map);
 
-  setupIndoorsLocationDataSource();
+  #ifdef PERMISSIONS_PLATFORM
+    requestBluetoothPermision();
```

Comment:
Not sure if we want a better name on these or not. I only say that because we aren't just checking bluetooth permissions w/ this function, we're checking bluetooth and then location permissions. I'd say the same for the `requestLocationPermission` function. It requests location then sets up ILDS.

Maybe something like `requestBluetoothThenLocationPermissions` or something else. If you don't agree you don't have to change anything.

Reply:
Sounds good, modified the names